src/parsing/lpLexer.ml

```diff
@@ -103,44 +103,6 @@ type token =
 
 exception SyntaxError of strloc
 
-(** Lexer for Lambdapi syntax. *)
-module Lp_lexer : sig
-
-  val is_uid : string -> bool
-  (** [is_uid s] is [true] iff [s] is a regular identifier. *)
-
-  val is_invalid_bindlib_id : string -> bool
-  (** [is_invalid_bindlib_id s] is [true] iff [s] ends with a positive integer
-     with leading zeros. *)
-
-  val pp_uid : string Base.pp
-  (** [pp_uid s] prints the uid [s], in escaped form if [s] is not a regular
-   identifier. *)
-
-  val is_identifier : string -> bool
-  (** [is_identifier s] is [true] iff [s] is a valid identifier. *)
-
-  val is_keyword : string -> bool
-  (** [is_keyword s] returns [true] iff [s] is a keyword. *)
-
-  val is_debug_flag : string -> bool
-  (** [is_debug_flag s] is true iff [s] is a debug flag. *)
-
-  val lexer : lexbuf -> unit -> token * Lexing.position * Lexing.position
-  (** [lexer buf] is a lexing function on buffer [buf] that can be passed to
-      a parser. *)
-
-  val equiv : string
-  (** [equiv] is the name of the symbol "≡" in unification rules. *)
-
-  val cons : string
-  (** [cons] is the name of the symbol ";" in unification rules. *)
-
-  val unif_rule_path : Path.t
-  (** [unif_rule_path] is the path for [equiv] and [cons].
-      It cannot be entered by a user. *)
-
-end = struct
```

Comment:
I'd prefer to keep the signature, I think it really helps to know what can be used outside and what cannot.

Reply:
It's not very nice to disconnect declarations, comments and regexp definitions. Probably one day we should add .mli files.

Reply:
We can already provide some interfaces right now.